manifests/plugin/abrt/params.pp

```diff
@@ -1,6 +1,7 @@
 # Default parameters for the Pulp smart proxy plugin
 class foreman_proxy::plugin::abrt::params {
   $enabled                 = true
+  $version                 = 'installed'
```

Comment:
@ekohl do you mean remove $version from abrt::params, add `plugin_version = 'installed'` to the main params.pp and in the class just do `class foreman_proxy::plugin::abrt($version = $foreman_proxy::plugin_version)`, to save re-defining it?